io.otsaloma.nfoview.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.otsaloma.nfoview
+runtime: org.gnome.Platform
+runtime-version: 3.30
```

Comment:
Should be `runtime-version: "3.30"`...like _stringified_.

Reply:
Done, same for the git tag version.